xarray/test/test_dataset.py

```diff
@@ -3186,6 +3186,23 @@ def test_full_like(self):
         self.assertEqual(expect['d2'].dtype, bool)
         self.assertDatasetIdentical(expect, actual)
 
+    def test_combine_first(self):  # works just like xr.merge([self, other])
```

Comment:
I would move this comment below, to just above the code block where you compare to the result of using `merge`